creator-cards/naweed-akram.json

```diff
@@ -1,26 +1,23 @@
-﻿{
+{
   "name": "Naweed Akram",
-  "sectionTitle": null,
-  "countryCode": "bh",
-  "slogan": "",
   "type": "CREATOR",
-  "bio": "",
-  "tags": ".NET MAUI",
-  "socials": {
-    "youtube": "https://www.youtube.com/@naweedakram",
-    "linkedin": "https://www.linkedin.com/in/naweed/",
-    "twitter": "https://twitter.com/xgeno",
-    "github": "",
-    "mastodon": "",
-    "sessionize": ""
-  },
+  "tags": ".NET, C#, .NET MAUI, Xamarin, Mobile",
+  "bio": "Solution Architect, Developer/Evangalist, Certified Project Management Professional (PMI-PMP), Enterprise Architect (TOGAF 9). Like to help community.",
+  "countryCode": "bh",
+  "github": "https://github.com/naweed",
   "channels": [
     {
       "name": "Blog",
       "url": "https://blogs.xgenoapps.com/"
     }
   ],
-  "feeds": [
+  "socials": {
+    "youtube": "https://www.youtube.com/@naweedakram",
+    "linkedin": "https://www.linkedin.com/in/naweed/",
+    "twitter": "https://twitter.com/xgeno",
+    "github": "https://github.com/naweed"
+  },
+  "projects": [
     {
       "type": "LinkTagRssType",
```

Comment:
This is an old type we no longer use, the template was updated @naweed 

can you please set the type to either "dotnet" or "personal"

Personal is required if your feed contains unrelated content (like personal, social, soft skills etc anything not dev related)

Reply:
@DeeJayTC 
Updated the json file. Hope this is fine now.

Thanks